src/StackAdmin/AzureStack/AzureStack.psd1

```diff
@@ -57,7 +57,7 @@
         @{ModuleName = 'Azs.Backup.Admin'; RequiredVersion = '0.3.0'; },
         @{ModuleName = 'Azs.Commerce.Admin'; RequiredVersion = '0.2.0'; },
         @{ModuleName = 'Azs.Compute.Admin'; RequiredVersion = '0.2.1'; },
-        @{ModuleName = 'Azs.Fabric.Admin'; RequiredVersion = '0.3.0'; },
```

Comment:
Don't need to change this.

Reply:
Talked to Bala, the change is needed.  I keep searching for Azs.Storage.Admin instead of Azs.Fabric.Admin